packages/website/src/theme/MDXComponents/RuleAttributes.tsx

```diff
@@ -1,4 +1,3 @@
-/* eslint-disable @typescript-eslint/no-unsafe-assignment */
```

Comment:
[Non-Actionable] Heh, I wonder how much this will come into conflict with #9339...